subgraphs/bridgeworld-stats/src/mappings/pilgrimage.ts

```diff
@@ -13,11 +13,10 @@ import {
 
 export function handlePilgrimagesStarted(event: PilgrimagesStarted): void {
   const params = event.params;
-  const tokenIds = params._ids1155;
   const tokenAmounts = params._amounts1155;
 
   let pilgrimagesCount = 0;
-  for (let i = 0; i < tokenIds.length; i++) {
+  for (let i = 0; i < tokenAmounts.length; i++) {
```

Comment:
This change doesn't really matter as the ids and amounts array will always be the same length.

Reply:
That's what I figured, but better to make it explicitly pulling from the `tokenAmounts` here